.github/workflows/conformance-gateway-api.yaml

```
@@ -60,11 +104,7 @@ jobs:
       - name: Set image tag
         id: vars
         run: |
-          if [ ${{ github.event.pull_request }} ]; then
-            SHA=${{ github.event.pull_request.head.sha }}
-          else
-            SHA=${{ github.sha }}
-          fi
+          SHA=${{ inputs.SHA || github.sha }}
```

Can we standardize to the same as other workflows?
```suggestion
          if [ "${{ github.event_name }}" = "workflow_dispatch" ]; then
            SHA="${{ inputs.SHA }}"
          else
            SHA="${{ github.sha }}"
          fi
```
I know it's kinda stupid here since there is only one variable but I think it helps to keep a consistent structure.